feedback/feedback.py

```diff
@@ -248,65 +203,67 @@ def get_url(icon_type, i):
         if self.p_user < self.p:
```

Comment:
This condition can be simplified into one line.

Reply:
Yes, but I don't refactor code at all

Reply:
Done